src/main/java/hudson/plugins/parameterizedtrigger/FileBuildParameterFactory.java

```diff
@@ -161,9 +147,6 @@ public FormValidation doCheckEncoding(@QueryParameter String encoding) {
                 } catch(IllegalCharsetNameException e) {
                     return FormValidation.error("Bad Encoding Name");
                 }
-                if(!ParameterizedTriggerUtils.isSupportNonAsciiPropertiesFile()) {
-                    return FormValidation.warning("Non-ascii properties files are supported only since Java 1.6.");
-                }
```

Comment:
Seems to be reverting parts of #55, I guess because of the assumption that Java 6 is required, though not quite complete since the Java 5-related text in `help-encoding.html` was left.